app/src/main/res/values/dimens.xml

```diff
@@ -4,16 +4,20 @@
     <dimen name="activity_horizontal_padding">8dp</dimen>
     <dimen name="activity_vertical_padding">8dp</dimen>
     <dimen name="input_margin_start_to_end">8dp</dimen>
-    <dimen name="title_text_size">12dp</dimen>
+    <dimen name="title_text_size">16sp</dimen>
     <dimen name="margin_top_bottom">32dp</dimen>
+    <dimen name="controls_margin_horizontal">8dp</dimen>
     <dimen name="te_button_height">45dp</dimen>
     <dimen name="views_update_top_margin">100dp</dimen>
     <dimen name="views_update_bottom_margin">150dp</dimen>
-
-    <dimen name="input_margin_start_to_end">8dp</dimen>
-    <dimen name="activity_horizontal_margin">8dp</dimen>
-    <dimen name="activity_vertical_margin">8dp</dimen>
-    <dimen name="no_dimension">0dp</dimen>
```

Comment:
Not going to say anything about this.... but please tomorrow remind me about this line so I can explain something at training